scripts/install_prerequisites.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash 
+
+
+sudo apt-get update
+ 
+sudo apt-get install git 
+
+sudo apt-get install unzip
+ sudo apt-get install default-jre
+
+sudo apt-get install curl python-setuptools python-dev python-pip 
+sudo pip install --upgrade pip
```

Comment:
Ah, I don't mind it, but what does it do? I couldn't find the documentation for this flag.